### PR TITLE
DOCS-11370 credential formula

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-11370 credential formula

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/guided-setup.adoc [hunk 1]
```diff
@@ -58,10 +58,22 @@ If your Anypoint account has the organization administrator role, provide your A
 Otherwise, provide these app details:
 +
 .. Set *Client ID* to the client ID from the connected app you created.
-.. In your Anypoint Platform organization, click *Access Management* > *Identity Providers* > *SAML 2.0* > *Edit* and copy the *IDP Issuer* string.
-.. Set *IDP Issuer* to the *IDP Issuer* string.
-.. In your Anypoint Platform organization, click *Access Management* > *Identity Providers* > *SAML 2.0* > *Edit* and copy the *Audience* string.
-.. Set *Audience* to the *Audience* string.
+.. To find the identity provider ID, run this request:
```
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc:61]: Looking at the context for the PR, I see that this is a really long task, with only two first-level steps. This indicates some sort of organizational issue that should be addressed.

File: modules/ROOT/pages/manual-setup.adoc [hunk 2]
```diff
@@ -84,6 +84,9 @@ The following instructions use URLs for the US cloud that begin with `+https://a
 . Set *Token Endpoint Url* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+`.
 . Set *Issuer* to the client ID of the connected app you saved previously.
 . Set *Per User Subject* to the following formula.
```
Comment (kbostrom) [inline modules/ROOT/pages/manual-setup.adoc]: Where is "the following formula"?

File: modules/ROOT/pages/manual-setup.adoc [hunk 3]
```diff
@@ -84,6 +84,9 @@ The following instructions use URLs for the US cloud that begin with `+https://a
...
++
+The guided setup uses a slightly different formula in the `IDP_ISSUER` field, but either formula works.
++
```
Comment (kbostrom) [inline modules/ROOT/pages/manual-setup.adoc]:
```suggestion
. Set *Per User Subject* to one of the following formulas:
+

```

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/guided-setup.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc:61]: This task has the three first-level steps "Select your organization's Anypoint management plane...", "Create a connected app...", and "Authenticate and authorize the connected app...". However, three may still be too low. I will review the organization here in a separate Jira story.

File: modules/ROOT/pages/manual-setup.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/manual-setup.adoc]: These are the formulas on lines 94 and 99.

File: modules/ROOT/pages/manual-setup.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/manual-setup.adoc]: Thank you! Fixed.

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**